MTM-59434 Announcement about endpoint access changes in microservice SDK (#2192)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/cumulocity-1020-110-0-endpoints-access-change-in-microservices-SDK.md

---
date: 

> title: Restricting Access to Spring boot actuator endpoints

Reviewer: ```suggestion
title: Restricting access to Spring boot actuator endpoints
```

> title: Restricting access to Spring boot actuator endpoints

Reviewer: ```suggestion
title: Restricting access to Spring Boot actuator endpoints
```

product_area: Application enablement & solutions
change_type:
  - value: change-inv-3bw8e

>     label: Announcement 

Reviewer: Shouldn´t it better be "API change"?

Reviewer: @mgrumann Could you please take a look?

Author: After a discussion with @mgrumann , we decided to indeed change it to API change, as you proposed.

Author: As we have changed this point already, I am resolving the conversation to be able to merge.

component:
  - value: component-Sv2buFZ5I
    label: Microservice SDK
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-58316
version: 1020.110.0 
---

> Microservices SDK exposes certain spring boot actuator endpoints, including /loggers, and provides an initial security configuration regarding the exposed spring boot actuator endpoints. 

Reviewer: ```suggestion
The Microservice SDK exposes certain Spring boot actuator endpoints, including `/loggers`, and provides an initial security configuration regarding the exposed spring boot actuator endpoints. 
```

> The Microservice SDK exposes certain Spring boot actuator endpoints, including `/loggers`, and provides an initial security configuration regarding the exposed spring boot actuator endpoints. 

Reviewer: ```suggestion
The Microservice SDK exposes certain Spring Boot actuator endpoints, including `/loggers`, and provides an initial security configuration regarding the exposed Spring Boot actuator endpoints. 
```

> Due to security considerations, the POST-access to the /loggers endpoint has been restricted to the roles TENANT_ADMIN, TENANT_MANAGEMENT_ADMIN, APPLICATION_MANAGEMENT_ADMIN and APPLICATION_MANAGEMENT_UPDATE

Reviewer: ```suggestion
For security reasons, the POST access to the `/loggers` endpoint has been restricted to the roles TENANT_ADMIN, TENANT_MANAGEMENT_ADMIN, APPLICATION_MANAGEMENT_ADMIN and APPLICATION_MANAGEMENT_UPDATE.
```
